CHANGELOG.md

```diff
@@ -7,72 +7,109 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
```

Comment:
Remove the empty lines between titles and content. This makes git not complain about merge conflicts when pulling commits from the master branch.